app/routes/labs/route.tsx

```diff
@@ -26,7 +26,7 @@ export default function () {
     <>
       <Header />
       <h1>ACROSS</h1>
-      <main>
+      <main className="bg-ink text-white text-thin">
```

Comment:
How does this look without `text-thin`? White text on a black background already decreases legibility compared to black on white; sometimes we actually make text thicker when on a dark background to make it more readable.

```suggestion
      <main className="bg-ink text-white">
```

Reply:
I found thin more readable. YMMV.